content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -0,0 +1,92 @@
+# How to customize fonts in React Native
+
+![Main image](images/customize_fonts_main.png)
+
+## Introduction
+
+Like every newbie in Rootstrap, my first assigment was to develop Target MVD application, in my case, in React Native. While doing this, one of the first things I tried was to set the family font that the application uses, and I found out it wasn't as easy as I would expected.
+
+The biggest issue was that all the tutorials said to excecute "react-native link" command and after executing it, the application would stop building successfully.
+
+## Overview
+
+In this tutorial you will learn how to add and link fonts manually, without using the link command.
+
+First you will see all the configurations that need to be done in order to be able to use customize fonts and then you will learn its usage in a simple application.
+
+## Configuration
+
+### Create Project
+
+First of all, you should create a project in react native. To do that, initiate terminal in the folder where you want to create your project and excecute:
+
+https://gist.github.com/ManuViola77/50605886d8c4f2184ea4316cec255b80
+
+### Install libraries
+
+After creating the project, there are some libraries that need to be installed:
+
+https://gist.github.com/ManuViola77/ce547c1d8241a21e536c773a80a6f97e
+
+### Download and rename fonts
+
+Since this is a tutorial about customizing fonts, you will need to have some fonts downloaded, in order to add them to your React Native project.
+
+There are many places where you can get this customized fonts. For this tutorial, I used two different fonts, [GoodFeelingSans](https://www.wfonts.com/font/good-feeling-sans) and [Dan'sDisney](https://www.fontspace.com/dans-disney-font-f24536). In both cases I ended up with a .ttf file.
+
+For IOS is important the font's filename, because it should be the same as its PostScript name. To know this, you should open your .ttf file with Font Book, see what its PostScript name is and rename the font to it.
+
+![Font Book](images/customize_fonts_postscript_name.png)
+
+### Configurations in React Native
+
+There are some steps that need to be done in order to use your custom fonts in a React Native project. There are also special configuration depending if you want to use android or IOS.
+
+#### General
+
+In both cases, you will need to create in your project's root this folders: **src/assets/fonts**. Inside the fonts folder, you should add you custom fonts .ttf files, one for every font you want to use.
+
+Also, modify **babel.config.js** adding this code:
+
+https://gist.github.com/ManuViola77/37e02b2a1da60c8178a67dd1194a941f
```

Comment:
Here too. I don't really see the point on having a link just to see a few lines of code